src/component/feature/feature-list-item-component.jsx

```diff
@@ -14,6 +14,7 @@ const Feature = ({
     metricsLastHour = { yes: 0, no: 0, isFallback: true },
     metricsLastMinute = { yes: 0, no: 0, isFallback: true },
     revive,
+    updateable,
```

Comment:
This one is never set to any value as far as I can see.

Reply:
I forgot about adding the PermissionComponent here instead of the prop.